src/models/activityReport.js

```diff
@@ -26,7 +26,8 @@ export default (sequelize, DataTypes) => {
         as: 'collaborators',
       });
       ActivityReport.belongsTo(models.Region, { foreignKey: 'regionId', as: 'region' });
-      ActivityReport.hasMany(models.File, { foreignKey: 'activityReportId', as: 'activityFiles' });
+      ActivityReport.hasMany(models.File, { foreignKey: 'activityReportId', as: 'attachments' });
+      ActivityReport.hasMany(models.File, { foreignKey: 'activityReportId', as: 'otherResources' });
       ActivityReport.belongsToMany(models.Goal, {
```

Comment:
We have to do this twice due to how sequelize works. This lets us use a where clause in the activityReport service